CRM/Custom/Form/MoveField.php

```diff
@@ -91,7 +91,15 @@ public function preProcess() {
    */
   public function buildQuickForm() {
 
-    $customGroup = CRM_Core_PseudoConstant::get('CRM_Core_DAO_CustomField', 'custom_group_id');
+    $customGroup = [];
+    $groups = \Civi\Api4\CustomGroup::get()
```

Comment:
We get bitten by forgetting to pass `FALSE` here... but I think in this context checking permissions is ok. Anyone who doesn't have permission to see custom groups has no business using this form.

Reply:
TRUE

Thanks!